src/main/app/case/definition.ts

```diff
@@ -2453,18 +2454,22 @@ export interface hearingDay{
   hearingDate: string,
   startTime: string,
   amPm:string,
+  preparedStartTime:string,
   durationInDayOrHours:number,
   minutes:number,
+  hearingDurationText: string,
   judgeName:string | null | undefined,
   venue:string | null | undefined,
   address:string | null | undefined,
   roomId:string | null | undefined,
+  detailsBlock: any
```

Comment:
hearingToAttendDetails and use appropriate type

Reply:
done